profile.php

```diff
@@ -152,12 +153,13 @@
 					foreach ($product_array as $key => $value) {
 				?>
 						<div class="product-item">
-							<form method="post" action="?action=add&product_id=<?php echo $product_array[$key]["product_id"]; ?>">
+							<form method="post" action="profile.php?action=add&product_id=<?php echo $product_array[$key]["product_id"]; ?>">
 								<div class="product-image"><img src="<?php echo $product_array[$key]["image"]; ?>"></div>
 								<?php if (isset($_SESSION["user-id"])) { ?><div class="favorite-switch" style="--star-color:<?php if (in_array($product_array[$key]['product_id'], $_SESSION["favProducts"])) echo "red";
 																															else echo "black" ?>;" onclick="document.location='index.php?action=switch-favorite&product_id=<?php echo $product_array[$key]['product_id']; ?>'"></div><?php } ?>
 								<div class="product-tile-footer">
 									<div class="product-title"><?php echo $product_array[$key]["name"]; ?></div>
+									<div class="product-price"><?php echo "৳" . $product_array[$key]["price"]; ?></div>
```

Comment:
I thought amra decide korechilam price na dilei better profile e? Cause ekhane to just favourite product save kore rakha purpose silo... ekhan theke to kinbe na so price rakha weird.. Ei jonnei to ketesilen... Ekhn abar add korlen keno?